lib/Marlin/Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -58,8 +63,16 @@ static inline void MINDA_BROKEN_CABLE_DETECTION__END() {}
 
 #include "../../lcd/ultralcd.h"
 
-#if HAS_DRIVER(L6470)                         // set L6470 absolute position registers to counts
-  #include "../../libs/L6470/L6470_Marlin.h"
+#if ENABLED(EXTENSIBLE_UI)
+  //#include "../../lcd/extui/ui_api.h"
```

Comment:
Why this is commented out?

In case this file doesn't exist I would rather the whole part of the patch not applied. That way it becomes updated later also with that file present.

Reply:
Unfortunately EXTENSIBLE_UI exists both in old and new and the API has diverged quite a bit. I kept the ifdef as it does apply cleanly in more cases, but I had to comment 3 places where ExtUI is used.